Explain what chained script interpreter construct is (#527)
NixOS/nix.dev: source/tutorials/reproducible-scripts.md

curl https://github.com/NixOS/nixpkgs/releases.atom | xml2json | jq .
```

> The first line is a standard hashbang. The additional hashbang lines are a Nix specific construct. The rest of the file is taken to be bash code because of the `-i bash` parameter.

Reviewer: ```suggestion
The first line is a standard shebang.
The additional shebang lines are a Nix-specific construct.
```

> We specify the `bash` program as the interpreter with the `-i` option.

Reviewer: ```suggestion
We specify `bash` as the interpreter for the rest of the file with the `-i` option.
```
